qt/scientific_interfaces/Indirect/ContainerSubtraction.cpp

```diff
@@ -521,5 +529,19 @@ IAlgorithm_sptr ContainerSubtraction::convertToHistogramAlgorithm(
   return convert;
 }
 
+IAlgorithm_sptr ContainerSubtraction::addSampleLogAlgorithm(
+    MatrixWorkspace_sptr workspace, const std::string &name,
+    const std::string &type, const std::string &value) const {
+  IAlgorithm_sptr shiftLog =
+      AlgorithmManager::Instance().create("AddSampleLog");
+  shiftLog->initialize();
+  shiftLog->setProperty("Workspace", workspace);
+  shiftLog->setProperty("LogName", name);
+  shiftLog->setProperty("LogType", type);
+  shiftLog->setProperty(
+      "LogText", boost::lexical_cast<std::string>(m_uiForm.spShift->value()));
```

Comment:
You are passing in a value into the function, but are not using it and setting the value from the ui form, why?

Reply:
This is also the cause of the compiler warning